source/html/visualizer/src/containers/bing.ts

```diff
@@ -1,32 +1,42 @@
-import { HostContainer} from "./host-container"; 
-import * as Adaptive from "microsoft-adaptivecards"; 
- 
-export class BingContainer extends HostContainer { 
-    static backgroundColor: string = "#fff"; 
- 
-    private _width: number; 
- 
-    constructor(width: number, styleSheet: string) { 
-        super(styleSheet); 
- 
-        this._width = width; 
-    } 
- 
-    protected renderContainer(renderedCard: HTMLElement): HTMLElement { 
-        var element = document.createElement("div"); 
-        element.style.width = this._width + "px"; 
-        element.style.backgroundColor = BingContainer.backgroundColor; 
-        element.style.overflow = "hidden"; 
- 
-        renderedCard.style.height = "100%"; 
- 
-        element.appendChild(renderedCard); 
- 
-        return element; 
-    } 
+import { HostContainer } from "./host-container";
+import {
```

Comment:
Why do this instead of what was before? Is it just to not have to type "Adaptive." everywhere?

Reply:
Basically only import minimal required object, it's easier for Webpack to do the code splitting work.